### DISCUSSION TITLE
Data-Driven 조직 문화 구축에 대해 조언해주세요

### DISCUSSION THREAD

**Discussion by jsaang** [playinpap/awesome-data-and-analytics-governance#11] [Q&A]
데이터에 기반한 의사결정이 잘 이루어지지 않는 조직에서 근무하고 있습니다. 어떻게 해야 Data-Driven한 문화를 만들어갈 수 있을까요?

**Comment by jongikp** [comment]
안녕하세요. 질문 주신 부분에 명확한 답변일지는 모르겠지만, 생각을 남깁니다.

제 생각에는 문화는 한순간에 이루어진다고 생각하지 않습니다.
따라서 잘 만들어나가는 creator와 잘 퍼트리는 influencer 의 조합이 중요하다고 생각합니다.

data 전담 부서는 creator 로서 데이터 기반 의사결정을 이룰 수 있는 infra를 구축해야한다고 생각합니다.
즉, 언제든 데이터 기반 의사결정을 할 수 있게끔 기반을 만들어두어야한다고 생각합니다.

제가 생각하는 기반은 다음과 같습니다.
- 퀄리티 높은 데이터
- 사용하기 쉬운 도구
- 지속 가능하며 정확한 리포팅

먼저, 퀄리티 높은 데이터를 구축하기 위해서는 데이터 생산 ~ 가공 단계를 촘촘하게 구축해야한다고 생각합니다.
raw data(db or client log) 부터 정확하고 사용성 높은 데이터를 생산하여 저장하고, (ingestion)
저장된 데이터를 잘 처리할 수 있는 공간으로 옮겨오고, (engineering)
그러한 공간에서 사용자들이 쓰기 쉽게 처리해두는 것 (modeling)
그리고 이러한 전체적인 흐름이 문제가 없는지 모니터링 하는 것 (monitoring)
이라고 생각합니다.

그리고 사용하기 쉬운 도구는 예를 들어, Google BigQuery 와 같은 클라우드 데이터 웨어하우스 및 시각화 도구 들이라고 생각합니다.
데이터를 사용하는 사람들이 실제로 데이터를 사용하는 경험에 있어서, 느리고 복잡하고 어렵다면 접근하기 정말 어려울 것이라고 봅니다.
따라서 이러한 툴들을 적절하게 활용하는 것이 중요하다고 생각합니다.

끝으로 지속 가능하며 정확한 리포팅은 data 전담 조직의 분석가들이 조직의 데이터를 바탕으로 인사이트를 도출하고 제공하는 것들을 꾸준히 정확하게 전달을 하여, 우리 조직의 데이터에 대해서 신뢰하고 각각이 취한 액션들에 대한 의미를 생각할 수 있도록 해야한다고 봅니다.
이 부분은 조직 내에 한명에게라도 도움이 된다면 반드시 해야하는 것이라고 생각하고, 이러한 부분들이 누적되었을 때 더 빛을 낸다고 생각합니다.

그리고 influencer 선정에 있어서 조직 내의 전파력 혹은 영향력이 있는 분을 타겟으로 잡아야하지 않을까 생각합니다.
이 영역은 [지속 가능하며 정확한 리포팅] 과 매우 밀접하다고 생각합니다. 전파력이 있고 영향력이 있는 사람은 대체로 높은 직책을 가진 분들일 것입니다.
그 분들에게 우리 조직의 데이터에 대해 꾸준히 전달하고, 그 분들이 관심을 갖고 직접 조회해보고 자료를 공유하고 하는 모습을 보여주신다면, 따라하지 않을 구성원들은 거의 없을 것이라고 생각합니다. 즉, [지속 가능하며 정확한 리포팅]을 그러한 분들을 target으로 한다면 보다 빠르게 데이터에 기반한 의사결정을 하는 문화가 전파되지 않을까 생각듭니다.

요약드리면, 양질의 데이터가 흐를 수 있도록 환경을 만드는 것과 데이터를 쉽고 빠르게 활용하여 영향력 있는 분들을 대상으로 꾸준히 공유한다면 전보다 나은 Data-Driven한 문화가 되지 않을까 생각듭니다.

화이팅하시길 바랍니다! 💪
